esx_service/cli/vmdkops_admin.py

```diff
@@ -515,6 +529,14 @@ def status(args):
     print("LogLevel: {0}".format(log_config.get_log_level()))
 
 
+def edit_vol(args):
+    edit_ok = vmdk_ops.edit_vol_opts(args.volume, args.options) 
+    if edit_ok:
+        print('Successfully updated settings for : {0}'.format(args.volume))
```

Comment:
printing message here seems unneeded.  Do we do it in other places ? I think we do need a log record though .